### DISCUSSION TITLE
Linux drag file over Air app crash

### DISCUSSION THREAD

**Discussion by pol2095** [airsdk/Adobe-Runtime-Support#1902] [Q&A] [answered]
Hello,

when I drag a file (icon) over an Air app, the Air app crash ?

Now it doesn't work, but Air on Linux supports [drag-in](https://help.adobe.com/en_US/as3/dev/WS5b3ccc516d4fbf351e63e3d118666ade46-7d83.html) and [drag-out](https://help.adobe.com/en_US/as3/dev/WS5b3ccc516d4fbf351e63e3d118666ade46-7d84.html) gesture ?

Thanks

**Comment by pol2095** [comment]
adt return this messsage :
```
Application crashed with an unhandled SIGSEGV
Crashlog has been dumped in /tmp/airCrashLogs/1027_1124_UZT2a5
```

**Reply by ajwfrost** [reply]
Hi - we're not able to reproduce this, can you confirm which version you're using, and which OS/version, and if you've got any specific ActionScript code in there which may be needed in order to cause the error?

thanks

**Comment by pol2095** [comment, marked as answer]
I tried on Ubuntu 22.04 and Fedora 37, not ActionScript code is necessary
I created a bug issue #2243 (with an image)
